tests/test_layers.py

```diff
@@ -187,7 +189,7 @@ def train_k_lip_model(
     np.random.seed(42)
     # create the keras model, defin opt, and compile it
     model = generate_k_lip_model(layer_type, layer_params, input_shape, k_lip_model)
-    optimizer = Adam(lr=0.001)
+    optimizer = Adam(learning_rate=0.001)
```

Comment:
maybe `Adam(0.001)` would do the job when changing tensorflow version

Reply:
I've updated this to avoid warning of deprecated argument, After discussions, as the arg name `learning_rate` exists since TensorFlow 2.0, it should be ok for all supported TF versions in deel-lip.